planner/cascades/transformation_rules.go

```diff
@@ -130,6 +130,9 @@ var TiKVLayerOptimizationBatch = TransformationRuleBatch{
 	memo.OperandTopN: {
 		NewRulePushTopNDownTiKVSingleGather(),
 	},
+	memo.OperandWindow: {
+		NewRuleMergeAdjacentWindow(),
```

Comment:
Why this rule set into the `TiKVLayerOptimizationBatch`?

Reply:
> Why this rule set into the `TiKVLayerOptimizationBatch`?

My fault. I will adjust it to `TiDBLayerOptimizationBatch`.